components/lifter.py

```diff
@@ -1,63 +1,99 @@
 import ctre
+import wpilib
 
 
 class Lifter:
-    lift_motor: ctre.WPI_TalonSRX
+    motor: ctre.WPI_TalonSRX
```

Comment:
Please have a more descriptive name for this variable.

Reply:
Also this does not currently exist in `Robot`, so will not be injected properly.

Reply:
`Robot.lifter_motor` does exist.